framework/spree/utils/get-media-gallery.ts

```diff
@@ -0,0 +1,30 @@
+// Based on https://github.com/spark-solutions/spree2vuestorefront/blob/d88d85ae1bcd2ec99b13b81cd2e3c25600a0216e/src/utils/index.ts
+
+import type { ProductImage } from '@commerce/types/product'
+import type { SpreeProductImage } from '../types'
+
+const getMediaGallery = (
+  images: SpreeProductImage[],
+  getImageUrl: (
+    image: SpreeProductImage,
+    minWidth: number,
+    minHeight: number
+  ) => string | null
+) => {
+  return images.reduce<ProductImage[]>((productImages, _, imageIndex) => {
+    const imageUrl = getImageUrl(images[imageIndex], 800, 800)
```

Comment:
I saw this is based on https://github.com/spark-solutions/spree2vuestorefront/blob/d88d85ae1bcd2ec99b13b81cd2e3c25600a0216e/src/utils/index.ts. However it could help renaming `imageUrl` to `url` to then pass it as a named property in the return.

(not needed for a merge)

Reply:
👋 Shortened: https://github.com/spree/nextjs-commerce/pull/21